jsonnet/rules.libsonnet

```diff
@@ -381,6 +381,9 @@ function(params) {
           },
           labels: {
             severity: 'warning',
+            // All OpenShift alerts should have a namespace label.
+            // See: https://issues.redhat.com/browse/MON-939
+            namespace: 'openshift-monitoring',
```

Comment:
Shall we add a namespace to other alerts in this file, too?  for example the previous alert "ClusterMonitoringOperatorReconciliationErrors".

Reply:
This PR adds the namespace label to alerts that we know are missing them. In most cases that means adding a static label like this, because the query used in the rule is cross namespace, or can't be grouped by namespace for some other reason. The other alerts will already include a namespace label because the queries produce results with one. So, no need to change anything in most cases.